docs/extending-locust.rst

```diff
@@ -28,13 +28,43 @@ Event listeners can be registered at the module level in a locust file. Here's a
 Adding Web Routes
 ==================
 
-Locust uses Flask to serve the web UI and therefore it is easy to add web end-points to the web UI. 
+Locust uses Flask to serve the web UI and therefore it is easy to add web end-points to the web UI.
 Just import the Flask app in your locustfile and set up a new route::
 
     from locust import web
-    
+
     @web.app.route("/added_page")
     def my_added_page():
         return "Another page"
 
 You should now be able to start locust and browse to http://127.0.0.1:8089/added_page
+
+
+Enabling SSL
+==================
+
+The Locust web interface also supports SSL encrypted connections. This is especially useful
+if, for instance, you're testing an API which may incidentally expose personal information
+or application secrets through the web interface.
```

Comment:
@justiniso , @cgoldberg : Our use case was to run locust distributed across several AWS EC2 instances. In such a case, the provisioning of SSL and nginx would have additional complexity. I understand if you guys want to keep locust insecure by default and let everyone have to add their own security as you guys may not have the time to maintain these features.

To answer your previous question, sensitive information can appear in the paths being used which are displayed in the UI. For instance, URL encoded forms that may be used for authentication (not that I used this, but there could be APIs that do use it).

I see value in these features specifically because we had a real-life client specifically ask for them (multi-billion dollar corp).